src/status_im/contexts/chat/messenger/messages/list/view.cljs

```diff
@@ -161,6 +161,7 @@
                  :color               cover-bg-color
                  :icon                (if muted? :i/muted :i/activity-center)
                  :on-press            (fn []
+                                        (rf/dispatch [:dismiss-keyboard])
```

Comment:
Hi @ibrkhalil, Thank you very much for finding and fixing this issue.
Please can we add this `:dismiss-keyboard` to `:show-bottom-sheet` itself?

Reply:
Thanks Parvesh
That's an excellent point, As the bottom sheet and the keyboard won't look nice together in any case.